runtime/khala/src/lib.rs

```diff
@@ -252,6 +270,28 @@ construct_runtime! {
 pub struct BaseCallFilter;
 impl Contains<Call> for BaseCallFilter {
     fn contains(call: &Call) -> bool {
+        if let Call::PolkadotXcm(xcm_method) = call {
+            match xcm_method {
+                pallet_xcm::Call::execute { .. }
+                | pallet_xcm::Call::teleport_assets { .. }
+                | pallet_xcm::Call::reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_teleport_assets { .. } => {
+                    return false;
```

Comment:
looks no need `;`?

Reply:
We have `return` here, i think it's better to keep `;` at the end, even through compiler has not thrown error when i removed it.

Reply:
Better write like:

return match ... {
   a | b => true,
   c => false
}

Reply:
Done